presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -2,10 +2,23 @@
 JDBC Driver
 ===========
 
-Presto can be accessed from Java using the JDBC driver.
-Download :maven_download:`jdbc` and add it to the class path of your Java application.
+The Presto JDBC (Java Database Connectivity) driver allows users to access 
+Presto using Java-based applications. Both client- and server-side 
+applications, such as those used for reporting and database development, use 
+the JDBC driver. The JDBC driver is an API that provides methods to perform 
```

Comment:
This last sentence isn't really correct. The driver is an implementation of an API. JDBC is the actual API. Also, while JDBC supports update, Presto does not.

I see three scenarios for a user landing on this page:

* They've never heard of JDBC. Why are they here, and will this information be useful to them?
* They know they need a Presto JDBC driver for the application. Knowing about the JDBC API doesn't seem useful to them. They don't care, they just need the driver.
* They are already familiar with JDBC. In which case, this doesn't help

We could compromise by making the first "JDBC" reference in the first sentence a link to the Wikipedia page.

Reply:
It was mentioned in the definitive guide, so I brought it up here. Will remove it.